tonic/models/vic/netcdf2vic.py

```diff
@@ -37,25 +39,27 @@ def _run(args):
 
     for i, fname in enumerate(files):
         d = read_netcdf(os.path.join(paths['in_path'], fname),
-                        verbose=verbose)
-
+                        verbose=verbose)[0]
+        print(i)
         if i == 0:
 
             # find point locations
-            xs = d['xc']
-            ys = d['yc']
+            xs = d['lon']
+            ys = d['lat']
             posinds = np.nonzero(xs > 180)
             xs[posinds] -= 360
             print('adjusted xs lon minimum')
 
+	    
             for y, x in pyzip(yi, xi):
                 active_flag = False
                 for key in var_keys:
                     if (d[key][:, y, x].all() is np.ma.masked) \
                             or (mask[y, x] == 0):
                         active_flag = True
-                if not active_flag:
-                    point = (ys[y, x], xs[y, x])
+                if not active_flag:	  
+                    point = (ys[y], xs[x])
+		    print(point)
```

Comment:
This change was required because my original code was assuming the xs/ys were 2d. If we used `meshgrid` c. line 49, we could insure that the xs/ys variables were 2d.